### PR TITLE
Fit the back-cast first admission date with mitigation in mind

### LINKED ISSUE #439: ["model"] Index day fitting against prescribed doubling time ignores mitigation
### Summary

When the user provides a doubling time, and the model estimates a date of first hospitalization based on that and current census, it ignores that mitigation may have changed the curve prior to the forecast date. This is a hangover from when mitigation was always assumed to be implemented on the forecast date.

Originally noted https://github.com/CodeForPhilly/chime/pull/418/files#r402476603

### PR DESCRIPTION
**Pull request by PhilMiller**
Fixes: #439

### CODE REVIEW

**Comment by PhilMiller** [comment]
Passes existing tests, and seems to behave reasonably. I'm not in a mindset to construct good new tests for this at the moment, so would welcome others pushing additional commits here to do so.

File: src/penn_chime/models.py [hunk 1]
```diff
@@ -64,21 +64,40 @@ def __init__(self, p: Parameters):
...
+                logger.info('Set i_day = %s', i_day)
+            else:
+                projections = {}
+                best_i_day = -1
+                best_i_day_loss = float('inf')
+                for i_day in range(p.n_days):
+                    self.i_day = i_day
+                    raw = self.run_projection(p, self.gen_policy(p))
+
+                    # Don't fit against results that put the peak before the present day
```
Comment (mdbecker) [inline src/penn_chime/models.py:86]: This seems to be a no-op? Can we get rid of it?

**Comment by mdbecker** [comment]
Tested this out locally and it seems to work as expected compared against the live version 👍

File: src/penn_chime/models.py (hunk 1 above)
Comment (PhilMiller) [reply src/penn_chime/models.py:86]: Play with the parameters more - those cases do come up

Comment (mdbecker) [reply src/penn_chime/models.py:86]: Sorry totally misread this. LGTM

**Review by mdbecker — APPROVED** [review]

**Pull request merged**